### ISSUE TITLE
(Blog) Design: Typography of intro texts

### ISSUE THREAD

**Issue by esreverse** [satellytes/satellytes.com#441]
This was part of https://github.com/satellytes/satellytes.com/issues/353 but is still unsolved, so I created a new issue:

Introtexts: The first textblocks on every page should be 20px. The editor should be able to achieve that. It's already working e.g. on the service page, but I would like to make it available also in blog articles in markdown. The idea is that the first paragraph is an intro to the upcoming page/article so it should have larger (20px) text. Let's discuss (maybe also with Georgios) what idea he has to make larger floating text available in blog articles.

**Comment by georgiee** [comment]
Can you please support your issue with an image outlining the issue? Makes it much more accessible than text. The blog area and content are technically different places so it might be a different.

I would like to know how your feedback applies to some actual blog post example. Can you link an example and describe the issue with a screenshot? We can then deliver it. Ideally it's 100% aligned with the content, so we can say "whenever content starts we have some "intro text" that we already named "Section Header" in the past (I think ?)

Maybe it's a good idea to establish a common understand for the content and vocabulary so it get easier to talk about it. The Figma naming does not work for the actual development semantics.

**Comment by esreverse** [comment]
If you look at our blog posts, all of the start with an opening/intro/overview that informs the user of what he will learn. I really like that and i would emphasise this also visually. Plus: I could use this text in LinkedIn and Twitter already uses the first lines of the article in the teaser image. Maybe it's worth discussing the goal before starting.

https://satellytes.com/blog/four-ways-to-improve-collaboration-in-your-team/
The first two lines below the title should be 20px

https://satellytes.com/blog/boosting-developer-productivity-with-github-actions/
Here the first four lines should be 20px

spiegel.de is doing something similiar:
https://www.spiegel.de/kultur/musik/kanye-west-grammys-verzichten-auf-auftritt-a-f87247bb-7e74-4097-aba9-2ad74a3720c3

zeit.de also:
https://www.zeit.de/politik/ausland/2022-03/energiepolitik-wasserstoff-forschung-unternehmen-arabische-emirate-robert-habeck

cnet.com
https://www.cnet.com/culture/entertainment/oscars-2022-predictions-will-dune-sweep-the-board/

![EricsShot 2022-03-21 11 51 53](https://user-images.githubusercontent.com/6918281/159247255-a12db8f1-2c79-4b5d-bf1a-826f746500aa.png)
![EricsShot 2022-03-21 11 54 42](https://user-images.githubusercontent.com/6918281/159247288-17c69e55-6473-47eb-9a08-e30c6b189282.png)

**Comment by milljoniaer** [comment]
Since we are now getting our blog posts from contentful, what do you think about an extra optional rich text field for blog posts in contentful, which we call "introText" or something like that, which is displayed on the top of each blog post and with larger text?

That way we are backwards compatible and do not have to change old blog posts (but of course can) and can use this also for the fcb case

**Comment by esreverse** [comment]
I like the idea. And I could even quickly copy-paste the first paragraphs to this intro field for all old blog posts. Maybe a matter of minutes in contentful.

**Comment by esreverse** [comment]
Can and do we want to use this intro text somehow when sharing to other platforms? @georgiee mentioned something months ago if i'm not mistaken
